example_audio/src/ofApp.cpp

```diff
@@ -1,11 +1,12 @@
 #include "ofApp.h"
 
 void ofApp::setup() {
```

Comment:
Should there be a `ofLogToConsole` call here so that we can see errors/warnings/info be printed on the console?